### PR TITLE
feat(design-system): use suggestion and suggestion group from x-tailwindcss

### PR DESCRIPTION
**Pull request by tiborux**
EX-7492

### CODE REVIEW

**Review by herrardo — COMMENTED** [review]
@cris-crash I would like to review this one with you to understand some things better.

**Review by ajperezbau — CHANGES_REQUESTED** [review]
Good job 👌

File: src/components/my-history/custom-my-history.vue [hunk 1]
```diff
@@ -54,15 +54,17 @@
         </div>
```
[the remark this answers is not in the document]
Comment (tiborux) [reply src/components/my-history/custom-my-history.vue:54]: image

[the remark this answers is not in the document]
Comment (ajperezbau) [reply src/components/my-history/custom-my-history.vue:54]: I think that what she is saying is that the picture on the left is the better 🤣

**Review by herrardo — CHANGES_REQUESTED** [review]
Some things that we are seeing because of this PR:

1. Change My History and Clear All buttons from ghost to tight, to align with suggestions
image

2.Align Top Clicked with History Queries title 
image

3. Align delete icon in suggestions to the right in desktop, mobile, and my history as Alicia proposed
image
image

4. Change suggestions to lg size **only in mobile**
5. Change My History Suggestion padding from padding-04 to padding-02, top&bottom

6. With the new suggestions, we have lost the alignment of elements in mobile between the input text and the suggestions:

Before
image

After
image
 
Review performed by @herrardo and @cris-crash  :)

File: src/components/my-history/custom-my-history.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (tiborux) [reply src/components/my-history/custom-my-history.vue:54]: im retarded sorry

**Review by herrardo — CHANGES_REQUESTED** [review]
- The gap between the icon and the suggestion in Popular Searches is to wide, it should be as in history queries 


@cris-crash is coming to you with this @tiborux

File: src/components/my-history/custom-my-history.vue [hunk 2]
```diff
@@ -62,15 +62,17 @@
...
```
Comment (ajperezbau) [inline src/components/my-history/custom-my-history.vue:62]: Should the close buttons be aligned with the switch?

image

File: src/components/predictive-layer/predictive-layer.vue [hunk 3]
```diff
@@ -9,13 +9,17 @@
         v-slot="{ identifierResult }"
```
Comment (ajperezbau) [inline src/components/predictive-layer/predictive-layer.vue:9]: It seems that the query suggestions are not being highlighted.

image

**Review by ajperezbau — CHANGES_REQUESTED** [review]
Good job 👌

**Review by ajperezbau — APPROVED** [review]
Good job 👌

**Review by cris-crash — APPROVED** [review]

**Review by ajperezbau — APPROVED** [review]

**Review by herrardo — APPROVED** [review]

**Pull request merged**